js_modules/dagster-ui/packages/ui-core/src/partitions/DimensionRangeWizard.tsx

```diff
@@ -53,23 +141,26 @@ export const DimensionRangeWizard = ({
               health={health}
               setShowCreatePartition={setShowCreatePartition}
               isDynamic={isDynamic}
+              disabled={selectState !== 'custom'}
             />
           )}
         </Box>
-        {isTimeseries && (
+        {isTimeseries && !showQuickSelectButtons && (
```

Comment:
Oh hmm, I was sort of thinking that showQuickSelectButtons=false would hide these. 

Maybe `showQuickSelectButtons` could be `showAllQuickSelectOptions` or `showQuickSelectOptionsForStatuses` or something like that, that implies that with it set to false you still get Latest and All?

Reply:
Yeah there are two variants. The existing version and the new version with the buttons above.